### PR TITLE
Expanded README with more context

### PR DESCRIPTION
**Pull request by awanlin**
Added more context about the intentions behind the demo site, what's been added and customized as well as handy links.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,107 @@
...
+- Tech Radar: <https://demo.backstage.io/tech-radar>
+- To Do: <https://demo.backstage.io/catalog/default/component/backstage-demo/todos>
+
+## Code Customization
+
+We have made some code customization as well. The following sections go into detail about them.
+
+### Custom Theme
+
+We have created a custom theme called Aperture to help showcase what some of the possibilities a custom theme can allow and to have a working example.
```
Comment (freben) [inline README.md:44]:
```suggestion
We have created a custom theme called Aperture to help showcase what some of the possibilities are that a custom theme can allow and to have a working example.
```

**Review by freben — APPROVED** [review]
All optional, preapproving

File: README.md [hunk 2]
```diff
@@ -1,5 +1,107 @@
...
+
+The automation is done using a GitHub workflow but the concepts and most of its parts could easily be transferred to other CI/CD systems like Azure DevOps Pipelines, CircleCI, etc. The parts of this workflow can be found here:
+
+- Core workflow: <https://github.com/backstage/demo/blob/master/.github/workflows/version-bump.yml>
+- Related scripts: <https://github.com/backstage/demo/blob/master/scripts/set-release-name.js>
+
+## Dependency Updates
+
+As Backstage uses packages for its various dependencies these will need to be kept up to date. This is terribly time consuming to do by hand thus we use [Renovate](https://github.com/renovatebot/renovate?tab=readme-ov-file#why-use-renovate) to help automate this process. Renovate will create Pull Requests for the various dependencies and all we need to to is review and merge them!
```
[the remark this answers is not in the document]
Comment (awanlin) [reply README.md]: Sure, I've adjusted this 👍

File: README.md [hunk 3]
```diff
@@ -1,5 +1,107 @@
...
+6. You should now see the more advanced Home, clicking on the "EDIT" button is a good place to start playing around with the features
+
+More details on the Home plugin can be found in the [Backstage homepage - Setup and Customization](https://backstage.io/docs/getting-started/homepage) documentation.
+
+### Search
+
+The Search plugin has been expanded to include results from the Explore plugins Tools list. This has been done in two places:
+
+1. Frontend changes are here: <https://github.com/backstage/demo/blob/master/packages/app/src/components/search/SearchPage.tsx>
+2. Backend changes are here: <https://github.com/backstage/demo/blob/dd624d915f0cb4f330e17dec91bdc8f50a784189/packages/backend/src/index.ts#L27>
```
[the remark this answers is not in the document]
Comment (awanlin) [reply README.md]: That makes sense to me, done 👍

File: README.md [hunk 4]
```diff
@@ -1,5 +1,107 @@
...
+
+The Aperture custom theme code can be found in [`aperture.ts`](https://github.com/backstage/demo/blob/master/packages/app/src/theme/aperture.ts)
+
+More details on creating a custom theme can be found in the [Creating a Custom Theme](https://backstage.io/docs/getting-started/app-custom-theme#creating-a-custom-theme) documentation
+
+### Home Plugin
+
+Beyond the installation of the Home plugin to get the most out of it you need to setup the initial code for it.
+
+You can find the code here: <https://github.com/backstage/demo/blob/master/packages/app/src/components/home/HomePage.tsx>
```
[the remark this answers is not in the document]
Comment (awanlin) [reply README.md]: I've gone through and found some ways to have less of these URLs in the text 👍

File: README.md (hunk 1 above)
Comment (awanlin) [reply README.md]: Fixed 👍

File: README.md [hunk 5]
```diff
@@ -1,5 +1,107 @@
...
+
+We have also setup the more advanced feature of the Home plugin that allows you to customize it more easily, the code for that is here: <https://github.com/backstage/demo/blob/master/packages/app/src/components/home/CustomizableHomePage.tsx>
+
+To see the more advanced Home in the Demo site you will need to do the following:
+
+1. Go to the Settings area here: <https://demo.backstage.io/settings>
```
[the remark this answers is not in the document]
Comment (awanlin) [reply README.md]: Cleaned this up too 👍

**Pull request merged**